plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml

```diff
@@ -74,6 +74,12 @@ items:
     verbs:
     - update
   - apiGroups:
+    - authorization.k8s.io
```

Comment:
@liggitt yeah, that was actually what I meant, but I see the double meaning of my comment.
I meant that this change (PR) is needed in the apiserver for kubeadm to work.

Thanks anyway!